binding/src/main/java/se/fortnox/reactivewizard/binding/AutoBindModules.java

```diff
@@ -27,37 +28,38 @@
      * We only want to find classes defined by our platform and user code.
      */
     private static final String[] PACKAGE_BLACKLIST = {
-        "-com.google",
-        "-liquibase",
-        "-io.netty",
-        "-org.yaml",
-        "-com.fasterxml",
-        "-org.postgresql",
-        "-org.apache",
-        "-org.hibernate",
-        "-rx",
-        "-org.jetbrains",
-        "-com.intellij",
-        "-com.netflix",
-        "-io.reactivex",
-        "-com.sun",
-        "-com.codahale",
-        "-com.zaxxer",
-        "-io.github.lukehutch",
-        "-io.prometheus",
-        "-org.jboss",
-        "-org.aopalliance",
-        "-redis",
-        "-net.minidev",
-        "-org.slf4j",
-        "-com.ryantenney",
-        "-net.logstash",
-        "-META-INF",
-        "-jar:java-atk-wrapper.jar",
-        "-jar:rt.jar",
-        "-jar:idea_rt.jar"
+        "com.google",
+        "liquibase",
+        "io.netty",
+        "org.yaml",
+        "com.fasterxml",
+        "org.postgresql",
+        "org.apache",
+        "org.hibernate",
+        "rx",
+        "org.jetbrains",
+        "com.intellij",
+        "com.netflix",
+        "io.reactivex",
+        "com.sun",
+        "com.codahale",
+        "com.zaxxer",
+        "io.github.lukehutch",
```

Comment:
Should this be io.github.classgraph now?

Reply:
good catch, fixed